.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      xvfb: # X virtual framebuffer
```

Comment:
Can we directly run `xvfb` similar to the framework spec rather than installing Selenium-related stuff that we don't plan to use?